packages/application-shell/src/constants.ts

```diff
@@ -1,7 +1,7 @@
 export const DIMENSIONS = {
   header: '43px',
   navMenu: '64px',
-  navMenuExpanded: '200px',
+  navMenuExpanded: '254px',
```

Comment:
It seems this value should be `245` since that's the one we use in the navbar:
![image](https://user-images.githubusercontent.com/97907068/204234834-8ae5f22b-76a8-43c9-b268-666fdf132725.png)

Otherwise, the overlay does not cover all the expected space:
![image](https://user-images.githubusercontent.com/97907068/204235075-43fca45b-cf1e-4e57-b746-43de78905100.png)

Reply:
Good catch!! I mixed up the numbers 🙈

Reply:
Fixed here: https://github.com/commercetools/merchant-center-application-kit/pull/2894/commits/0df8c27bae683c743b197493e7a4f62119845519